scripts/release

```diff
@@ -6,6 +6,14 @@ set -e
 
 cd "$(dirname "$0")/.."
 
+# Remove previous build
+if [ -n "$(ls | grep 'build')" ]; then
+	rm -r build/
+fi
+
+# Upgrade setuptools to the latest version
+python3 -m pip install --upgrade setuptools
```

Comment:
What is the reason to add this?

Reply:
I noticed the the metadata `project_urls` weren't redenered correctly and that the tag `bdist_wheel (0.30.0)` was included. That led me to belive that the `setuptools` version is probably outdated. Since this happend to me as well, I decided to add this.

If you want to, I'll remove it.
Just running `pip install --upgrade setuptools` would be enough as well.

Reply:
This may unexpectedly change the user env, I would prefer not to do it, if it is OK for you

Reply:
Absolutely! In that case it might be best to remove the `wheel` and `twine` installs down below as well.